### PR TITLE
[DT-2277] feat: handle users out of the experiment scope

### PR DESCRIPTION
**Pull request by BohdanOne**
**Resolves**: [DT-2277](https://linear.app/prismic/issue/DT-2277/add-success-toast-when-push-ready)

### Description
The experiment target only specific group of users.
For those who are not in scope of the experiment, we want to show the new version.

### Checklist

- [ ] If my changes require tests, I added them.
- [ ] If my changes affect backward compatibility, it has been discussed.
- [ ] If my changes require an update to the CONTRIBUTING.md guide, I updated it.

### How to QA [^1]

[^1]:
	Please use these labels when submitting a review:
	:question: #ask:&ensp;Ask a question.
	:bulb: #idea:&ensp;Suggest an idea.
	:warning: #issue:&ensp;Strongly suggest a change.
	:tada: #nice:&ensp;Share a compliment.

### CODE REVIEW

File: packages/slice-machine/src/hooks/usePromptToCreateContentExperiment.ts [hunk 1]
```diff
@@ -6,5 +6,13 @@ export function usePromptToCreateContentExperiment(): UsePromptToCreateContentEx
   const variant = useExperimentVariant("slicemachine-prompt-to-create-content");
 
   // serve the experimental version to user with Telemetry deactivated
-  return { eligible: variant === undefined || variant?.value === "on" };
+  return {
```
Comment (xrutayisire) [inline packages/slice-machine/src/hooks/usePromptToCreateContentExperiment.ts:14]: 💡 #idea

I'm wondering, what about?

```suggestion
  return variant?.value !== "control"
```

Maybe less good to get when it's eligible?

**Review by xrutayisire — APPROVED** [review]
LGTM!

I left a comment, but I'm really not sure, so I let you decide 🙂

**Review by dani-mp — APPROVED** [review]
left it approved but opened a discussion in slack https://prismic-team.slack.com/archives/C02L3FN3AJK/p1724425903321349

File: packages/slice-machine/src/hooks/usePromptToCreateContentExperiment.ts (hunk 1 above)
Comment (BohdanOne) [reply packages/slice-machine/src/hooks/usePromptToCreateContentExperiment.ts:14]: make sense, will change

Comment (BohdanOne) [reply packages/slice-machine/src/hooks/usePromptToCreateContentExperiment.ts:14]: changed

**Pull request merged**